src/www/ui/api/Controllers/GroupController.php

```diff
@@ -92,4 +92,63 @@ public function createGroup($request, $response, $args)
     }
     return $response->withJson($returnVal->getArray(), $returnVal->getCode());
   }
+
+  /**
+   * Get users with their roles from a given group
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function getGroupMembers($request, $response, $args)
+  {
+    $userId = $this->restHelper->getUserId();
+    /** @var UserDao */
+    $userDao = $this->restHelper->getUserDao();
+    $groupMap = $userDao->getAdminGroupMap($userId, $_SESSION[Auth::USER_LEVEL]);
+    $res = null;
+
+    if (empty($groupMap)) {
+      $res = new Info(400, "You have no permission to manage any group.", InfoType::ERROR);
+    } else {
+      $groupId = null;
+
+      /**
+       * Let group be the default one , if there's no group provided in the url params. *
+       **/
+
+      if (isset($args['id'])) {
+        $groupId = intval($args['id']);
+      } else
+        $groupId = key($groupMap);
+
+      /**
+       * The query to get the list of users with corresponding roles from the group. *
+       **/
+
+      $dbManager = $this->dbHelper->getDbManager();
+
+      $stmt = __METHOD__ . "getUsersWithGroup";
+      $dbManager->prepare($stmt, "select user_pk, user_name, user_status, user_desc, group_user_member_pk, group_perm
+         FROM users LEFT JOIN group_user_member gum ON gum.user_fk=users.user_pk AND gum.group_fk=$1
+         ORDER BY user_name");
+
+      $result = $dbManager->execute($stmt, array($groupId));
+      $usersWithGroup = $dbManager->fetchAll($result);
+
+      /**
+       * Convert back fields [user_pk , group_user_member_pk ,group_perm ] from String to Integer*
+       **/
+      foreach ($usersWithGroup as $key => $record) {
+        $usersWithGroup[$key]["user_pk"] = intval($record);
```

Comment:
If using `User` model in the response, it can easily be fetched using `$userModel = $this->dbHelper->getUsers($userId)[0]`